### ISSUE TITLE
train accuracy showing > 100

### ISSUE THREAD

**Issue by elbanan** [radtorch/preview#5]
## Scenario:

Using `weighted_sampler=True`

```python
data_folder = '/content/data/Data/CXR/TRAIN'
T = A.Compose([A.Resize(128,128)])
TR = A.Compose([A.CenterCrop(700, 700), A.Resize(128,128), A.HorizontalFlip(p=0.5)]) #A.RandomSizedCrop((512,512), 64, 64) #A.CenterCrop(300, 300),  A.Rotate(limit=10), 
ds = ImageDataset(folder=data_folder, split={'valid':0.2, 'test':0.2}, WW=350, WL=50,out_channels=1, \
                  transform={'train':TR,'valid': T,'test': T}, normalize=True,\
                  batch_size=64, 
                 weighted_sampler=True, shuffle=False
                 )

m2 = Model(model_arch='vgg16', in_channels=ds.out_channels, out_classes=len(ds.classes), pre_trained=True, unfreeze_all=True)
# criterion2 = nn.CrossEntropyLoss(weight=ds.class_weights.to('cuda'))
criterion2 = nn.CrossEntropyLoss()

optimizer2 = optim.Adam(m2.model.parameters(), lr=0.0001)
clf2 = ImageClassifier(model=m2.model, dataset=ds, criterion=criterion2, optimizer=optimizer2)
```

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/17037319/156388868-1bc8e5da-0d74-45d4-a1e1-0fd565a3a664.png">

**Comment by elbanan** [comment]
Fixed in 1.0.1
